### ISSUE TITLE
Sound cue for the ratio is too big vs too small

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#48]
During sound design meeting today, and https://github.com/phetsims/ratio-and-proportion/issues/45 (and #9 generally).

What would it sound like to cue when the ratio is bigger than desired, vs smaller than desired. 

- One idea for the Vibrato sound would be to have the semi tone be lower for one, and higher for the other. Currently it is just off between 220 (base frequency) and 230
- Another idea could be to pan the incorrect sound onto the ear that corresponds to what side of the ratio should be fixed.

**Comment by BLFiedler** [comment]
Just to note this is tied to a potential visual design brought up in #42

**Comment by emily-phet** [comment]
another reason to have color/sound feedback different when you’re above vs below ratio is as you use the keyboard you could jump from just above/below to just below/above the success point. Currently, the auditory feedback may not be that different, which could be confusing.

**Comment by emily-phet** [comment]
Forgoing this. Closing.
